mesa/agent.py

```diff
@@ -13,6 +13,7 @@ def __init__(self, unique_id, model):
         """ Create a new agent. """
         self.unique_id = unique_id
         self.model = model
+        self.pos = None
```

Comment:
Is this required to make mypy happy? Otherwise I am not sure how I feel about this regarding models without any spatial dimension

Reply:
Yes, otherwise mypy complains that an `Agent` doesn't have a `pos` attribute.

Reply:
I think it is harmless to have a `pos` attribute in the Agent specification, given that it is optional (i.e. has a default value of `None`). Otherwise using `pos` without specifying it would be too ad-hoc.